src/main/scala/org/allenai/deep_qa/parse/Parser.scala

```diff
@@ -86,6 +87,12 @@ class StanfordParser extends Parser {
     parsed
```

Comment:
Something weird is happening here to make the `Parser` tests fail, but I can't see what it is - the types match up ...